app/components/exp-lookit-survey-consent/component.js

```diff
@@ -79,6 +79,8 @@ export default ExpFrameBaseComponent.extend({
         finish() {
             let formValid = this.validate();
             if (formValid) {
+                this.session.set('completedConsentFrame', true);
```

Comment:
Just to be clear, this is already in PR #352?

Reply:
Yeah... so I put this change in its own PR (#352), in case we wanted to get a quick fix for the data availability bug into production ASAP. And then I branched off of that branch to do make this change (flagging the response as containing survey consent), because I needed it to get the lookit-api to pick up the responses in the consent manager. But since we didn't put this change out on its own, there's really no reason for #352 anymore.